pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
I previously wrote http://nixos.org/nixpkgs/manual/#chap-cross documenting all 3 platforms these, including acknowledging the rampant confusion they tend to sow.

I could link back to that documentation in every derivation that uses these, but I suspect that would be a great many links. I hope that long term, `buildPlatform`, `hostPlatform` and `targetPlatform` become as expected as `stdenv`, so no one would bat an eye at them.